.gitignore

```diff
@@ -371,3 +371,5 @@ wheelhouse/*
 dist/*
 venv*/**
 *.swp
+
+gperftools
```

Comment:
Ignore the git submodule. In windows, this directory is created when updating the submodule. Adding to .gitignore to... ignore it.